simpleLooping/binaryListToString.py

```diff
@@ -15,3 +15,18 @@
 # Ex:
 #   binaryListToString([1, 0, 1, 1])       # return "1101"
 #   binaryListToString([0, 1, 1, 0, 0, 1]) # return "100110"
+from typing import List
+
+def binaryListToString(lst: List[int]) -> str:
+    for num in lst:
+        if num != 0 and num != 1:
+            return -1
+    
+    binaryStr = ""
+    for i in range(len(lst)):
+        if lst[i] == 0:
```

Comment:
Please refactor this conditional structure to 1 line. You can use "+=" operation and trinary operation (the syntactic sugar for conditional structure).

Reply:
got it.